cfn-resources/database-user/test/databaseuser.sample-template.json

```diff
@@ -0,0 +1,131 @@
+{
+  "AWSTemplateFormatVersion": "2010-09-09",
+  "Description": "Returns all database users that belong to the specified project. To use this resource, the requesting API Key must have the Project Read Only role. This resource doesn't require the API Key to have an Access List.",
```

Comment:
Is this file new Example? If so should we delete https://github.com/mongodb/mongodbatlas-cloudformation-resources/blob/master/examples/database-user/user.json ?

Reply:
As far as the https://github.com/mongodb/mongodbatlas-cloudformation-resources/blob/master/examples/README.md is suggesting, 

"These examples are deprecated and will soon be removed from the code base in favor of the test/project.sample-cfn-request.json and test/project.sample-template.yaml examples contain in the source folder for each Resource Type."
Following this comment I have kept the file in ./test/databaseuser.sample-template.json.
I will delete this https://github.com/mongodb/mongodbatlas-cloudformation-resources/blob/master/examples/database-user/user.json file

Reply:
Handled in [6d075bc20598253c789ce5f833954a06a5fe750a](https://github.com/mongodb/mongodbatlas-cloudformation-resources/pull/132/commits/6d075bc20598253c789ce5f833954a06a5fe750a)